Dangerfile

```diff
@@ -209,12 +225,12 @@ module VSPDanger
 end
 
 if $PROGRAM_NAME != __FILE__
-  VSPDanger::Runner.run.each do |output|
-    case output[:severity]
-    when :error
-      fail output[:message] # rubocop:disable Style/SignalException
-    when :warning
-      warn output[:message]
+  VSPDanger::Runner.run.sort.each do |result|
```

Comment:
Implemented `<=>` so that the Results are sortable, making sure we fail an error instead of suppressing it with a warning.